content/graphing/infrastructure/cloudfunctions.md

```diff
@@ -0,0 +1,125 @@
+
+## Installation
+
+The Cloud Functions dashboard requires no installation of its own, but it relies on three data sources that require their own installation:
+
+1. [Amazon Web Services integration][2] (required)
+
+	This integration is used to populate summary graphs and the main functions table. Install this integration and ensure that Lambda metrics are reporting in your account.
+
+	*Note*: Metrics in the Cloud Function UI are delayed ~10 minutes, as this is the default speed at which we poll AWS APIs. Reach out to support@datadoghq.com if you would like this rate to be decreased to 5 minutes.
+
+2. [AWS X-Ray integration][4] (optional)
+	This integration provides traces for Lambda functions in the function detail page.
+	
+	Install the X-Ray integration and add the following permissions to the policy document in your AWS/Datadog Role:
+	```
+	xray:BatchGetTraces,
+	xray:GetTraceSummaries
+	```
+	If using a Customer Master Key to encrypt traces, add the `kms:Decrypt` method to your policy where the Resource is the Customer Master Key used for X-Ray.
+
+	Recommended X-Ray setup:
+
+	- Navigate to the Lambda function in the AWS console you want to instrument. In the “Debugging and error handling” section, check the box to “Enable active tracing”. This turns on X-Ray for that function.
+
+	- Import the X-Ray SDK in your function, and patch all supported libraries. This automatically causes X-Ray to trace all AWS calls and other X-Ray supported integrations. See [an example of this in Python][5].
+
+	*Note*: Traces are delayed ~5 minutes, as this is the speed at which Datadog polls AWS X-Ray APIs.
+
+3. AWS CloudWatch Logs (optional)
+
+	Install this if you want to see logs from your Lambda functions in the function detail page. This will also populate the additional metrics such as Memory Used (avg) and Last Start in your functions table.
+
+	To enable this, refer to the [documentation for sending Lambda logs to Datadog][6].
+
+## Searching, Filtering, Sorting
+
+### Tagging
+
+Datadog tags your Lambda functions and metrics with relevant AWS metadata and custom tags.
+
+AWS Metadata:
+
+- account_id
+- executedversion
+- functionname
+- memorysize
+- region
+- resource
+- runtime
+
+Ensure you have the `lambda:List*` permission in your Datadog IAM policy in order to have these tags collected.
+
+Ensure you have the `tag:GetResources` permission in your Datadog IAM policy in order to have custom tags collected.
+
+### String Search
+
+Search for functions using flexible string search that matches substrings on function name, AWS Metadata, and custom tags.
+
+### Filtering
+
+Use the faceted search functionality along the left-hand side of the page to narrow down the functions that are in view. All AWS and custom tags are available to use as filters.
+
+### Selecting metrics on table
+
+You can check and uncheck metrics you would like to view on the functions table. By default, invocations, duration (avg), and errors are on by default.
```

Comment:
Move up from non-default to default. 

- Estimated Cost
- Memory Used (avg)
- Last Start

(In that order after errors)